### DISCUSSION TITLE
404 There isn't a GitHub Pages site here.  If you're trying to publish one, read the full documentation to learn how to set up GitHub Pages for your repository, organization, or user account.

### DISCUSSION THREAD

**Discussion by Mehwishmushtaq** [community/community#86769] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

404
There isn't a GitHub Pages site here.

If you're trying to publish one, [read the full documentation](https://help.github.com/pages/) to learn how to set up GitHub Pages for your repository, organization, or user account.   
Help me to resolve this error in React Website .....Url is below 
https://mehwishmushtaq.github.io/MTech/

**Comment by kamranzafar4343** [comment]
What is the name of your HTML file.

**Reply by kamranzafar4343** [reply]
It should be index.html. Follow this tutorial along and also if it looks helpful. Mark it as answer, it would be helpful for me.

**Reply by kamranzafar4343** [reply]
https://www.youtube.com/watch?v=p1QU3kLFPdg&pp=ygU7aG93IHRvIHVwbG9hZCB5b3VyIHdlYnNpdGUgb24gZ2l0aHViIHBhZ2VzIHN1cGVyIHNpbXBsZSBkZXY%3D

**Comment by kimevn** [comment]
How can you fix it?

**Comment by jfullstackdev** [comment]
it's now hosting,

![image](https://github.com/community/community/assets/47092464/d47b21d0-4c28-40ce-bbfa-5804d69016e8)
